apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V97__add_rda_phase_columns.sql

```diff
@@ -0,0 +1,9 @@
+/*
+ * Adds new columns to the FISS and MCS claims tables to store the phase number
+ * and phase sequence number for use in filtering claims.
+ */
+
+ALTER TABLE rda.fiss_claims ADD phase smallint;
```

Comment:
Should we turn these into NOT NULLs in PACA-717?

Reply:
Good question.  According to the `record_source.proto` file these values are optional in the RDA API.  We could default them to 0 or something if they are not populated there.  Also we'd definitely have to backfill the claim tables before doing so.

Reply:
Disregard the backfill part of my reply...

Reply:
Ah I see that it's also optional in the metadata table, https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V95__add_rda_record_source.sql, so never mind I thought RDA had made those fields required.